cypress/integration/login.js

```diff
@@ -1,40 +1,43 @@
+import LoginPage from '../objects/loginPage';
+import TimeLoggingPage from '../objects/timeLoggingPage';
+
+const loginPage = new LoginPage();
+const timeLoggingPage = new TimeLoggingPage();
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+
+        loginPage.visit();
+        loginPage.getValidationError().should('not.visible')
+        loginPage.getLoginbutton().click();
+        loginPage.getValidationError().should('be.visible')
+
     })
 
     const rolesArr= [['User', 1], ["Team Lead", 2], ['Manager', 5], ['Accountant', 5], ['Admin', 6]]
 
     for (let i=0; i<rolesArr.length;i++) {
 
-        it('Should be able to login with each role User', function () {
+        it(`Should be able to login with each role ${rolesArr[i][0]}`, function () {

```

Comment:
The word 'each' has now become obsolete, since you specify every role explicitly during every iteration :)